composer.json

```diff
@@ -10,20 +10,22 @@
     ],
     "require": {
         "jms/metadata": "~1.5",
-        "symfony/property-access": "~2.4",
+        "symfony/property-access": "~2.6||~3.0",
```

Comment:
the question here is did we want to maintain 2.6 after releasing 1.0 ? if not we should change it now.